libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -49,10 +49,37 @@ do {                                            \
                        OUTOFGEOFENCE)
 
 
+static uint8_t counter = 0;     // For Debuging
+
+
 AP_GPS_SBF::AP_GPS_SBF(AP_GPS &_gps, AP_GPS::GPS_State &_state,
-                       AP_HAL::UARTDriver *_port) :
+                       AP_HAL::UARTDriver *_port, uint8_t _asterx_setup) :
```

Comment:
If you introduce an enum for this the entire switch below goes away, and future areas can use switch instead of an if chain, as well as being clearer.

Reply:
I will change this one to an enum. That  would be better indeed